scripts/asr_language_modeling/ngram_lm/install_beamsearch_decoders.sh

```diff
@@ -11,5 +11,24 @@ rm -rf OpenSeq2Seq
 cd decoders
 # patch setup code to support the recent distutils
 sed -i 's/, distutils/, distutils\nimport distutils.ccompiler/g' setup.py
+
+cp /workspace/nemo/scripts/installers/setup_os2s_decoders.py ./setup.py
 ./setup.sh
+
+# install Boost package for KenLM
+wget https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.bz2 --no-check-certificate && tar --bzip2 -xf /workspace/nemo/decoders/boost_1_80_0.tar.bz2 && cd boost_1_80_0 && ./bootstrap.sh && ./b2 --layout=tagged link=static,shared threading=multi,single install -j4 || echo FAILURE
+export BOOST_ROOT=/workspace/nemo/decoders/boost_1_80_0
```

Comment:
All paths should be based on a single path like "/workspace/nemo" defined on the top and the rest should just concat the extra path to the main one.